homeassistant/components/qwikswitch/__init__.py

```diff
@@ -221,7 +221,7 @@ def callback_qs_listen(qspacket):
     @callback
     def async_start(_):
         """Start listening."""
-        hass.async_add_job(qsusb.listen, callback_qs_listen)
+        qsusb.listen(callback_qs_listen)
```

Comment:
Calling `hass.async_job` will call `qsusb.listen` from an executor thread, which causes a threading error.

It seems fine to call this directly from the event loop instead:
https://github.com/kellerza/pyqwikswitch/blob/9d4f080048221eaee93e3eefcf641919ff1af586/pyqwikswitch/async_.py#L74-L77

The code at https://github.com/kellerza/pyqwikswitch doesn't seem to be in sync with PyPi though, at least there is no 0.93 tag